src/ActionList/ActionList.features.stories.tsx

```diff
@@ -41,6 +42,109 @@ export const SimpleList = () => (
   </ActionList>
 )
 
+export const WithHeading = () => (
+  <ActionList>
+    <ActionList.Heading as="h1">Details</ActionList.Heading>
+    <ActionList.LinkItem href="https://github.com/primer/react#readme">
+      <ActionList.LeadingVisual>
+        <BookIcon />
+      </ActionList.LeadingVisual>
+      Readme
+    </ActionList.LinkItem>
+    <ActionList.LinkItem href="https://github.com/primer/react/blob/main/LICENSE">
+      <ActionList.LeadingVisual>
+        <LawIcon />
+      </ActionList.LeadingVisual>
+      MIT License
+    </ActionList.LinkItem>
+    <ActionList.LinkItem href="https://github.com/primer/react/stargazers">
+      <ActionList.LeadingVisual>
+        <StarIcon />
+      </ActionList.LeadingVisual>
+      <strong>1.5k</strong> stars
+    </ActionList.LinkItem>
+    <ActionList.LinkItem href="https://github.com/primer/react/watchers">
+      <ActionList.LeadingVisual>
+        <EyeIcon />
+      </ActionList.LeadingVisual>
+      <strong>21</strong> watching
+    </ActionList.LinkItem>
+    <ActionList.LinkItem href="https://github.com/primer/react/network/members">
+      <ActionList.LeadingVisual>
+        <RepoForkedIcon />
+      </ActionList.LeadingVisual>
+      <strong>225</strong> forks
+    </ActionList.LinkItem>
+  </ActionList>
+)
+
+export const WithHeadingInMenu = () => (
```

Comment:
**Semantics**

The html structure is matching with the usage without a menu. Do we want anything behave differently when the actionlist is in a menu?

**Styles**

The default styles I provided is a bit broken due to not having a left and top space - should this be on consumer's side?? ([storybook reference](https://primer-c4f9c0d65c-13348165.drafts.github.io/storybook/?path=/story/components-actionlist-features--with-heading-in-menu))

Reply:
> The default styles I provided is a bit broken due to not having a left and top space - should this be on consumer's side?

Nah, I definitely thing we should handle this ourselves